canonical-websites/www.ubuntu.com
# Update logo clouds to use links

## Done

- Update logo clouds to use links, if they exist
- Fix unconstrained image resizing with combo of max-width and max-height
- Filed an [issue to fix in Vanilla](https://github.com/vanilla-framework/vanilla-framework/issues/1605)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
  - http://0.0.0.0:8001/containers
  - http://0.0.0.0:8001/cloud/partners
  - http://0.0.0.0:8001/desktop/partners
  - http://0.0.0.0:8001/internet-of-things/gateways
  - http://0.0.0.0:8001/internet-of-things/robotics
  - http://0.0.0.0:8001/server/hyperscale
  - http://0.0.0.0:8001/server
- See that the logos have links
- See that the logos maintain their aspect ratio
- See that the logos do not get clipped on small screens

## Issue / Card

Fixes #2769
